model/checkininfo.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+// CheckInInfoWebConnectivity contains the array of URLs returned by the checkin API
+type CheckInInfoWebConnectivity struct {
+	URLs []URLInfo `json:"urls"`
```

Comment:
IIRC there is also the reportID we should be using. I would like to add this to the structure even though we are currently not using it?

Reply:
Let us put all in `model/checkin.go` so this can become `CheckInWebConnectivity`!